draft-ietf-mls-protocol.md

```diff
@@ -4426,6 +4426,11 @@ has to meet a specific set of requirements:
   init secret as described in {{external-initialization}}.
 * The sender type for the MLSAuthenticatedContent encapsulating the External Commit MUST be
   `new_member_commit`.
+* Normally, the joiner computes the interim transcript hash using the confirmed transcript
+  hash and the confirmation tag from the GroupInfo. The only exception is when the GroupInfo
+  was sent in an epoch with epoch ID 0 (i.e., before any members were added); in this case,
+  the confirmation tag in GroupInfo can be arbitrary and the joiner uses the empty interim
+  transcript hash.
```

Comment:
I think this is correct given the way group creation is specified right now.

Another way we could fix this is to make the 0->1 transition a little more like other transitions, in the following sense:

* Have the group creator choose an epoch secret (not just an init secret)
* Set the initial interim transcript hash to the MAC computed with the empty confirmed transcript hash and the confirmation key from the initial epoch secret
* Then the external joiner computes the interim transcript hash as normal.

I have a slight preference for doing this latter fix.  It makes things more consistent, but would be a slightly larger change.